### ISSUE TITLE
Quoted strings may break the parser

### ISSUE THREAD

**Issue by mvasilkov** [RichardLitt/inline-style-2-json#6]
This issue is similar to #1.

Consider the following:

```
<p style="
    background: url('foo.bmp; display: none;')
">
```

This will produce a `{display: none}` rule in the output, while it clearly should not.

**Comment by RichardLitt** [comment]
Interesting! I don't have the time to fix this at the moment; I'd be happy to look at a PR, though.

**Comment by RichardLitt** [comment]
I added a failing test in 59965e081c0957788e5d01e5278482329661e7d5. You're right this should work.

**Comment by RichardLitt** [comment]
Actually, I'm not sure. I think that [`<url>`](https://developer.mozilla.org/en-US/docs/Web/CSS/url) can't have semicolons in it, which I think is actually breaking this, not the strings.

**Comment by mvasilkov** [comment]
Well it's not just the url, there are other string props in CSS, like content, and font-family.

**Comment by RichardLitt** [comment]
Yeah, but I don't think it was the strings that were the issue, but the semicolons and colons in the string.

**Comment by RichardLitt** [comment]
Btw, @mvasilkov - happy to accept a PR for this with relevant tests.
